src/ilcdlib/epd/reader.py

```diff
@@ -620,12 +628,6 @@ def to_openepd_epd(
         program_operator = (
             program_operator_reader.to_openepd_org(lang, base_url, provider_domain) if program_operator_reader else None
         )
```

Comment:
Invoke `get_verifiers` and group verifications by 2 groups: `internal` and external. Internal should go under ext, the first item from external goes to `third_party_verifier`